packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.cnstyles.ts

```diff
@@ -24,6 +24,11 @@ const getItemHighContrastStyles = memoizeFunction(
           borderColor: 'Highlight',
           color: 'HighlightText',
           MsHighContrastAdjust: 'none',
+          selectors: {
```

Comment:
can you add this style to `subMenuIcon: { ... }` block below instead targeting the icon using className?

Reply:
Thanks @xugao I've moved the change down to `subMenuIcon`

Reply:
Thanks!